factory/post.js

```diff
@@ -26,7 +25,10 @@ function processContent(data) {
     var item = entries[i],
         postTitle = item.title.$t,
         content   = item.content.$t,
-        url       = item.link[4].href,
+        // Do not assume the link will always remain item 4 in the array
+        url       = _.filter(item.link, function (link) {
+          return link.rel === 'alternate' && link.type === 'text/html';
+        })[0].href,
```

Comment:
In that case we will query Google 😀 

On a more serious note, it does not make sense for them to do that. If you already have an alternate that is text/html, adding another alternate that is text/html does not make sense. In any case, we will take the first text/html alternate link.